src/cubism-common/MotionManager.ts

```diff
@@ -230,20 +388,55 @@ export abstract class MotionManager<Motion = any, MotionSpec = any> extends util
         }
 
         let audio: HTMLAudioElement | undefined;
+        let analyzer: AnalyserNode | undefined;
+        let context: AudioContext | undefined;
 
         if (config.sound) {
+            const isBase64Content = sound && sound.startsWith('data:audio/wav;base64');
+            if(sound && !isBase64Content){
+                var A = document.createElement('a');
+                A.href = sound;
+                sound = A.href; // This should be the absolute url
+                // since resolveURL is not working for some reason
+            }
+            const isUrlPath = sound && (sound.startsWith('http') || sound.startsWith('blob'));
             const soundURL = this.getSoundFile(definition);
-
+            let file = soundURL;
             if (soundURL) {
+                file = this.settings.resolveURL(soundURL) + "?cache-buster=" + new Date().getTime();
```

Comment:
It's not our responsibility to invalidate the cache. If a user wants to invalidate the cache, they can do it themselves.

```suggestion
                file = this.settings.resolveURL(soundURL);
```